Engine/Settings.cpp

```diff
@@ -1305,7 +1305,7 @@ Settings::initializeKnobsPlugins()
     _pluginsTab->setName("plugins");
 
 #if defined(__linux__) || defined(__FreeBSD__)
-    std::string searchPath("/usr/OFX/Plugins");
+    std::string searchPath("/usr/share/OFX/Plugins");
```

Comment:
wrong, see http://openeffects.org/documentation/reference/ch02s02.html

Reply:
mb, change was meant to be excluded